scripts/scripts.js

```diff
@@ -0,0 +1,57 @@
+const cards = document.querySelectorAll('.memory-card');
+
+let hasFlippedCard = false;
+let firstCard, secondCard;
+let lockBoard = false;
+
+
+function flipCard() {
+      if (lockBoard) return;
+      if (this === firstCard) return;
+      this.classList.add('flip');
+
+
+      if (!hasFlippedCard) {
+            hasFlippedCard = true;
+            firstCard = this;
+            return;
+      }
+
+      secondCard = this;
+      hasFlippedCard = false;
+
+      checkForMatch();
+
+}
+
+function checkForMatch() {
+      let isMatch = firstCard.dataset.food == secondCard.dataset.food;
```

Comment:
use `===` instead of `==` whenever possible.

`==` forces a type coercion and can mess with equality checks.

Reply:
Done